### ISSUE TITLE
Widget not reaching full width grid. worse with ListWidget

### ISSUE THREAD

**Issue by mark-yacoub** [sAleksovski/react-native-android-widget#22]
So I tried it on a physical device and things aren't great.
without the listwidget, things are in good order as shown in the following screenshot 
![image](https://github.com/sAleksovski/react-native-android-widget/assets/11547193/5f465f2c-d89d-4147-9a1a-4e37cbcf62de)

however, you can see that the widget is always not reaching the edge of the widget border.
Here is my xml
```
<?xml version="1.0" encoding="utf-8"?>
<!-- https://saleksovski.github.io/react-native-android-widget/docs/tutorial/register-widget#add-a-widget-provider-xml-file -->
<appwidget-provider xmlns:android="http://schemas.android.com/apk/res/android"
    android:targetCellWidth="3"
    android:targetCellHeight="2"

    android:updatePeriodMillis="1860000"

    android:initialLayout="@layout/rn_widget"

    android:previewImage="@drawable/widget_screenshot"
    android:description="@string/widget_description"

    android:resizeMode="horizontal|vertical"

    android:widgetCategory="home_screen">
</appwidget-provider>
```

here is my container
```
  return (
    <FlexWidget style={styles.container}>
      {renderDate()}
      {!!tasksForDate.length && renderTasks()}
      {!tasksForDate.length && renderNoTasks()}
    </FlexWidget>
  );
};

const styles = {
  container: {
    backgroundColor: '#ffffff',
    borderRadius: 16,
    flex: 1,
    height: '130',
    overflow: 'auto',
    paddingVertical: 5,
    width: 'match_parent',
  },
```

currently, this is my renderTasks
```
  const renderTasks = () => tasksForDate.map(task => renderTask(task));
```
however, once i wrap in List widget like
```
  const renderTasks = () => (
    <ListWidget style={{height: 'match_parent', width: 'match_parent'}}>
      {tasksForDate.map(task => renderTask(task))}
    </ListWidget>
  );
```
it starts acting weird and it looks like
![image](https://github.com/sAleksovski/react-native-android-widget/assets/11547193/fb3e1604-a769-4764-9518-fc55052cf65a)

scrolling is great though.

_Originally posted by @mark-yacoub in https://github.com/sAleksovski/react-native-android-widget/issues/17#issuecomment-1582788162_

**Comment by sAleksovski** [comment]
I just released `0.6.1`, trying to resolve some of these issues.
Now the widget might be cropped on some launchers that do not report the correct widget size, but touch will work correctly.

Can you try it to see if it fixes your issue?

**Comment by sAleksovski** [comment]
I'll close this issue since there was no response.
You can reopen it if the problem is still there.
